common/coap-server/src/main/java/org/thingsboard/server/coapserver/TbCoapDtlsSettings.java

```diff
@@ -51,6 +56,9 @@ public class TbCoapDtlsSettings {
     @Value("${transport.coap.dtls.bind_port}")
     private Integer port;
 
+    @Value("${transport.coap.dtls.retransmission_timeout}")
```

Comment:
add default value:

```
@Value("${transport.coap.dtls.retransmission_timeout:9000}")
```

Reply:
This is required for backward-compatibility with old yml files.

Reply:
OK